syntax.php

```diff
@@ -168,7 +164,6 @@ function handle($match, $state, $pos, Doku_Handler $handler){
             }
 
             // parse it
-            require_once(dirname(__FILE__).'/XMLParser.php');
```

Comment:
Since this is part of the plugin and does not have a namespace, I don't think it will be autoloaded.

Reply:
fixed